app/Product.php

```diff
@@ -1,10 +1,9 @@
-<?php
-
-namespace App;
-
+<?php namespace App;
 use Illuminate\Database\Eloquent\Model;
 
 class Product extends Model
+
+
 {
```

Comment:
Opening brace of a class must be on the line following the class declaration; found 2 line(s)